Assets/Develop/Scripts/Familiar/FamiliarController.cs

```diff
@@ -18,8 +19,31 @@ public class FamiliarController : MonoBehaviour {
     private Vector3 worldTargetPos;
     private Vector2 defaultTarget;
 
+    [SerializeField]
+    private SkeletonMecanim skeletonMecanim;
+    [SerializeField]
+    private Animator animator;
+
+    [SerializeField]
+    private PlayerController controller;
+
     void Start () {
         player = transform.parent.gameObject;
+        controller = transform.parent.gameObject.GetComponent<PlayerController>();
+
+        skeletonMecanim = GetComponentInChildren<SkeletonMecanim>();
+        if(!skeletonMecanim) {
```

Comment:
```suggestion
        if(skeletonMecanim == null) {
```
to keep it consistent with `if(controller == null)` below.

Should also change `if(!animator)` below.

Reply:
Changed to be consistent